README.md

```diff
@@ -31,7 +31,7 @@ table).
 If this behavior is not what you want, look at `skipDuplicates` option
 - **If you are looking for glob, or sass like imports (prefixed partials)**,
```

Comment:
Doesn't look like your plugin supports SASS imports.
```suggestion
- **If you are looking for glob imports**,
```

Reply:
pardon my ignorance, as i havent used Sass in many years, 

https://sass-lang.com/guide#topic-4

i feel like the original wording's intention was to catch the attention of sass migrates that used partials,
thats how i stumbled across this

ill have another edit up in a second

Reply:
yeah did you read my newer commit?

i reworded the context

Reply:
Yes; it still mentions migrating from SASS partials; when your plugin does not support anything like SASS partials.

Reply:
oh i thought SASS's  @imports had glob functionality,
ill push another